lib/matplotlib/widgets.py

```diff
@@ -3408,7 +3421,8 @@ class PolygonSelector(_SelectorWidget):
     @_api.rename_parameter("3.5", "markerprops", "handle_props")
     @_api.rename_parameter("3.5", "vertex_select_radius", "grab_range")
     def __init__(self, ax, onselect, useblit=False,
-                 props=None, handle_props=None, grab_range=10):
+                 props=None, handle_props=None, grab_range=10,
+                 draw_box=False):
```

Comment:
Rename `draw_box` to `draw_bbox`?

Reply:
Add a `bbox_props` and `bbox_handle_props` to pass to `RectangleSelector`?

Reply:
Since this is user facing, I think `box` instead of `bbox` is better - most users won't have come across the concept of bounding boxes or the abbreviation `bbox`.

Good idea to pass through kwargs to RectangleSelector, I will do that.